packages/astro/test/units/dev/dev.test.js

```diff
@@ -43,89 +32,16 @@ describe('dev container', () => {
 		});
 	});
 
-	it('HMR only short circuits on previously cached modules', async () => {
```

Comment:
I don't understand why this fails but I feel like HMR behaviours shouldn't be unit tested. The entire HMR flow is very intricate and should be tested with a proper project fixture.